Assets/Editor/ExportOBJ.cs

```diff
@@ -0,0 +1,158 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.Collections;
+using System.IO;
+using System.Text;
+
+public class ObjExporterScript
+{
+	private static int StartIndex = 0;
+
+	public static void Start()
+	{
+		StartIndex = 0;
+	}
+	public static void End()
+	{
+		StartIndex = 0;
+	}
+
+	public static string MeshToString( MeshFilter mf, Transform t )
+	{
+		Vector3 s = t.localScale;
+		Vector3 p = t.localPosition;
+		Quaternion r = t.localRotation;
+
+		int numVertices = 0;
+		Mesh m = mf.sharedMesh;
+		if ( !m )
+		{
+			return "####Error####";
+		}
+		Material[] mats = mf.renderer.sharedMaterials;
+
+		StringBuilder sb = new StringBuilder();
+
+		foreach ( Vector3 vv in m.vertices )
+		{
+			Vector3 v = t.TransformPoint( vv );
+			numVertices++;
+			sb.Append( string.Format( "v {0} {1} {2}\n", v.x, v.y, -v.z ) );
```

Comment:
David, didn't you say it was faster to use += when concatenating strings? Obviously this code isn't really performant heavy but string.Append is used a bunch in here.